networking/route/pmtu/common/network.sh

```diff
@@ -0,0 +1,1815 @@
+#!/bin/bash
+# vim: sts=8 sw=8 noexpandtab:
+# This is for network operations
+
+trap 'cleanup_swcfg' HUP TERM KILL EXIT
+
+# ---------------------- Global variables  ------------------
+
+# variable for configuration files
+SWCFG_UNDO="/mnt/testarea/swcfg_undo.sh"
+if uname -r | grep 4.14
+then
+		NIC_INFO_URL=${NIC_INFO_URL:-http://pkgs.devel.redhat.com/cgit/tests/kernel/plain/networking/inventory/nic_info-alt}
```

Comment:
The URLs here and below shouldn't be present.

Reply:
the scripts need to be clean-up...some code need to be removed. I forgot to upload the clean one.. thanks for the catch.